modules/jupyter-widget/src/playground/create-deck.js

```diff
@@ -113,14 +113,14 @@ function createStandaloneFromProvider({
   // Common deck.gl props for all basemaos
   const handlers = handleEvent
     ? {
-        onClick: info => handleEvent('click', info),
-        onHover: info => handleEvent('hover', info),
-        onResize: size => handleEvent('resize', size),
+        onClick: info => handleEvent('deck-click-event', info),
```

Comment:
Note the new more specific event names as these now go over the generic transport back-channel, potentially shared with other features.

Reply:
The string constants here will need to be changed too: https://github.com/visgl/deck.gl/blob/master/bindings/pydeck/pydeck/widget/widget.py#L93